### PR TITLE
adding port-forward

### PR DESCRIPTION
**Pull request by puja108**

### CODE REVIEW

**Comment by puja108** [comment]
RFR @marians @oponder

**Comment by marians** [comment]
Comment about the older content in this guide: Neither the ingress manifest nor the `kubectl apply -f myingress.yaml` command contain a namespace. Unless the user want's to deal with the `default` ns, this could result in some more search. We can either add `namespace: <namespace>` to the manifest or `--namespace <namespace>` to the command.

**Comment by marians** [comment]
If port-forwarding is only for pods, that part doesn't really fit with the title and introduction:

> Accessing Services from the Outside
> Once you have a Service running ...

---

The structure is a bit odd now. The browser/certs part should be a sub-section of "Access any service from through the API proxy", but currently it's the other way around.

**Comment by puja108** [comment]
added the namespace to ingress, also tried to fix the structure, as for the title and intro I jsut added "pods" ;)

RFR @marians

**Comment by marians** [comment]
@puja108 Changes not pushed yet, I guess

**Comment by puja108** [comment]
now

File: content/guides/accessing-services-from-the-outside/index.md [hunk 1]
```diff
@@ -1,24 +1,91 @@
 +++
-title = "Accessing Services from the outside"
-description = "You can access services from outside your cluster either through the API proxy or through an Ingress."
-date = "2016-10-25"
+title = "Accessing Pods and Services from the outside"
+description = "You can access Pods and services from outside your cluster either through the API proxy or through an Ingress."
```
Comment (marians) [inline content/guides/accessing-services-from-the-outside/index.md]: Hmm. That sentence isn't really true, is it. How about "Here we explain how to access pods and services for testing purposes and make services available to the public, using Ingress, API proxies and port-forwarding."

File: content/guides/accessing-services-from-the-outside/index.md [hunk 2]
```diff
@@ -1,24 +1,91 @@
...
+Now you can access your Pod locally via `localhost:<local-port>`.
+
+### Access any Service from through the API proxy {#api-access}
+
+The Kubernetes API comes with an inbuilt proxy, which you can use to access Services deployed on your cluster. However, as your API is secured by your credentials you need to set up authentificated access to the API in your system to be able to access the API in your browser.
+
+Once you have set up the certificates on your system, you should be able to access any service in your cluster through following template (replace <fields> accordingly).
+
+```nohighlight
+https://api.<cluster-name>.k8s.gigantic.io/api/v1/proxy/namespaces/<namespace>/services/<service-name>:<port>/
```
Comment (marians) [inline content/guides/accessing-services-from-the-outside/index.md]: `<cluster-name>` should be `<cluster-id>`.

File: content/guides/accessing-services-from-the-outside/index.md [hunk 3]
```diff
@@ -1,24 +1,91 @@
...
+
+or to have it running in the background
+
+```nohighlight
+kubectl port-forward --namespace <namespace> $POD <local-port>:<pod-port> &
+```
+
+Now you can access your Pod locally via `localhost:<local-port>`.
+
+### Access any Service from through the API proxy {#api-access}
```
Comment (marians) [inline content/guides/accessing-services-from-the-outside/index.md]: This should be a h2 (`##`)

File: content/guides/accessing-services-from-the-outside/index.md [hunk 4]
```diff
@@ -1,24 +1,91 @@
...
+
+The Kubernetes API comes with an inbuilt proxy, which you can use to access Services deployed on your cluster. However, as your API is secured by your credentials you need to set up authentificated access to the API in your system to be able to access the API in your browser.
```
Comment (marians) [inline content/guides/accessing-services-from-the-outside/index.md]: `authentificated` -> `authenticated`

File: content/guides/accessing-services-from-the-outside/index.md (hunk 1 above)
Comment (puja108) [inline content/guides/accessing-services-from-the-outside/index.md]: changed to "Here we explain different ways by which you can access Pods and Services from outside your cluster."

File: content/guides/accessing-services-from-the-outside/index.md (hunk 4 above)
Comment (puja108) [inline content/guides/accessing-services-from-the-outside/index.md]: done

**Pull request merged**